client/gutenberg/extensions/map/edit.js

```diff
@@ -180,10 +188,10 @@ class MapEdit extends Component {
 						/>
 						<ButtonGroup>
 							<Button type="button" onClick={ this.updateAPIKey } isDefault>
-								{ __( 'Update Key' ) }
+								{ __( 'Update Token' ) }
 							</Button>
 							<Button type="button" onClick={ this.removeAPIKey } isDefault>
```

Comment:
Are we removing the "Remove token" button?

Reply:
No, it's still there: https://github.com/Automattic/wp-calypso/blob/a95de2425761522d220d94e2d9629acc4dabc856/client/gutenberg/extensions/map/edit.js#L193-L195